src/components/signup/components/SignupForm.js

```diff
@@ -47,8 +76,47 @@ class SignupModal extends React.Component {
                 onOk={onCreate}
                 className={CLASSNAME_SCROLLBAR}
             >
-            
-                <Form  >
+                <Form>
+                    <FormItem className={CLASSNAME_ITTEM1}>
+                        {PRODUCER}
+                        <Radio.Group 
+                            defaultValue={false}
+                            buttonStyle={SOLID}
+                            onChange={this.toggleIsProducer}
+                            size={SMALL}
+                            className={CLASSNAME_RADIOBUTTONS}
+                        >
+                            <Radio.Button value={true}>{YES}</Radio.Button>
+                            <Radio.Button value={false}>{NO}</Radio.Button>
+                        </Radio.Group>
+                    </FormItem>
+                    {isProducer &&
+                    <FormItem
+                        {...formItemLayout}
+                        label={PRODUCT_TYPE}
+                    >
+                        {
+                            getFieldDecorator('productType', {
+                                rules:
+                                [{ message: { MESSAGE }, required: true }],
```

Comment:
There is something wrong with this line that is causing your code to have errors. find it and fix it.